docker-codescene/start-codescene.sh

```diff
@@ -1,2 +1,4 @@
 #!/bin/bash
-java -XshowSettings:vm $JAVA_OPTIONS -XX:MaxRAMPercentage=60 -jar "/opt/codescene/codescene-enterprise-edition.standalone.jar" | tee -a /codescene/codescene.log
+mkdir -p $CODESCENE_ANALYSIS_RESULTS_ROOT
+mkdir -p $CODESCENE_CLOSED_REPOSITORIES_ROOT
```

Comment:
typo should be `CODESCENE_CLONED_REPOSITORIES_ROOT`

Reply:
The typo that just doesn't want to go away.... Glad you saw that!